docs/whats-new/dotnet-docs-mod0.md

```diff
@@ -1,147 +1,94 @@
+- [andrebaltieri](https://github.com/andrebaltieri) - Andre Baltieri ![There were 1 pull requests merged by Andre Baltieri.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [Applesauce314](https://github.com/Applesauce314) - Andrew Moskevitz ![There were 1 pull requests merged by Andrew Moskevitz.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [aydjay](https://github.com/aydjay) - Alex Jones ![There were 1 pull requests merged by Alex Jones.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [cdcd72](https://github.com/cdcd72) - Neil ![There were 1 pull requests merged by Neil.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [chIsmaeel](https://github.com/chIsmaeel) - Muhammad Ismaeel ![There were 1 pull requests merged by Muhammad Ismaeel.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [compujuckel](https://github.com/compujuckel) -  ![There were 1 pull requests merged by .](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [DaveKobrin](https://github.com/DaveKobrin) - David Kobrin ![There were 1 pull requests merged by David Kobrin.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [DickBaker](https://github.com/DickBaker) - Dick Baker ![There were 1 pull requests merged by Dick Baker.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [EnessenE](https://github.com/EnessenE) - Enes ![There were 1 pull requests merged by Enes.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [ermshiperete](https://github.com/ermshiperete) - Eberhard Beilharz ![There were 1 pull requests merged by Eberhard Beilharz.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [hmmwhatsthisdo](https://github.com/hmmwhatsthisdo) -  ![There were 1 pull requests merged by .](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [mathieu-benoit](https://github.com/mathieu-benoit) - Mathieu Benoit ![There were 1 pull requests merged by Mathieu Benoit.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [nhangeland](https://github.com/nhangeland) - Niklas ![There were 1 pull requests merged by Niklas.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [NN---](https://github.com/NN---) - NN ![There were 1 pull requests merged by NN.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [ShalokShalom](https://github.com/ShalokShalom) - ShalokShalom ![There were 1 pull requests merged by ShalokShalom.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [Smaug123](https://github.com/Smaug123) - Patrick Stevens ![There were 1 pull requests merged by Patrick Stevens.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [testfirstcoder](https://github.com/testfirstcoder) -  ![There were 1 pull requests merged by .](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
```

Comment:
It might be better to change the link text to something more adaptable like:

```suggestion
- [testfirstcoder](https://github.com/testfirstcoder) -  ![1 pull requests](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
```

Reply:
Great suggestion. Rather than hand-edit this, let's start that next month. PR is here: https://github.com/dotnet/docs-tools/pull/241